src/components/common/TmConnectedNetwork.vue

```diff
@@ -232,6 +284,48 @@ export default {
   background: var(--success);
 }
 
+.powered-hr {
+  display: block;
+  height: 1px;
+  border: 0;
```

Comment:
is this necessary?

Reply:
yeah, `hr` is another kind of beast. Actually it was quite hard to make it look the way it is now. 

I cannot find the SO thread that helped me anymore 😅